base.rkt

```diff
@@ -83,9 +83,9 @@
 ; number of times to loop a FLIF (0 = forever)
 (define image-num-loops 0)
 (define animation-thread (make-parameter #f))
-(define decoder-thread (make-parameter #f))
+;(define decoder-thread (make-parameter #f))
```

Comment:
If these actually aren't used anywhere (and I see they're not) we prolly wanna delete the commented-out lines (unless you're planning to revive them very soon).